src/debugger/nodeDebugWrapper.ts

```diff
@@ -43,6 +43,7 @@ interface ILaunchArgs {
     platform: string;
     target?: string;
     internalDebuggerPort?: any;

```

Comment:
@digeff This is a user provided value. We make the best effort to parse it, whether it is a string or a number. We used `any` because compilation fails if you pass anything other than string to parse.